src/datasets/fxa_metrics/funnels.md

```diff
@@ -0,0 +1,179 @@
+# Firefox Account Funnels
+
+There are two primary "funnels" that users step through when authenticating with FxA. The **registration** funnel reflects the steps required for a **new** FxA user (or more precisely, email address) to create an account. The **login** funnel reflects the steps necessary for an **existing** FxA user to sign into their account.
+
+We are also in the process of developing funnels for paying subscribers. We will add documentation on that once the work is is closer to complete.
+
+## Registration Funnel
+
+While there are some variations, the typical registration funnel is comprised of the steps described in the chart below. Except where noted, these events are emitted by the FxA content server.
+
+|Step|Amplitude Event|Flow Event|Description|
+|---|---|---|---|
+|1|`fxa_email_first - view`|`flow.enter-email.view`|View (impression) of the form that the user enters their email address into to start the process. Note that this form can be hosted by FxA, or hosted by the relying party. In the latter case, the relier is responsible for handing the user's email address off to the FxA funnel. See "counting top of funnel events" below.|
+|2|`fxa_reg - view`|`flow.signup.view`|View of the registration form. If the user got to this step via step 1, FxA has detected that their email address is not present in the DB, and thus a new account can be created. The user creates their password and enters their age.|
+|3|`fxa_reg - engage`|`flow.signup.engage`|A user focuses/clicks on one of the registration form fields.|
+|4|`fxa_reg - submit`|`flow.signup.submit`|A user submits the registration form (could be unsuccessfully).|
+|5|`fxa_reg - created`|`account.created`|This event is emitted by the auth server. It indicates that user has entered a valid email address and password, and that their account has been created and added to the DB. However, the account is still "unverified" at this point and therefore not accessible by the user.|
+|6|`fxa_email - sent` (`email_type` = `registration`)|`email.verification.sent`|An email is sent to the user to verify their new account. Depending on the service, it either contains a verification link or a verification code that the user enters into the registration form to verify their email address.|
```

Comment:
In between 6 and 7, we also now have `fxa_reg - cwts_view/engage/submit` too now, right?

Reply:
(and there's also the `cwts_back` event, which kind of complicates the funnel because it can introduce repetition which might seem weird if your're not looking for that event in the flow)

Reply:
thanks for the review! added the cwts steps to the reg funnel